### PR TITLE
chore: fb app events v1 tests

### PR DESCRIPTION
**Pull request by sandeepdsvs**
## What are the changes introduced in this PR?

Updated fb proxy tests to v1 format

## What is the related Linear task?

Resolves INT-1481

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3141/test-report.html

File: test/integrations/destinations/fb/dataDelivery/business.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            app_user_id: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+            custom_events:
+              '[{"_logTime":1567333011693,"_eventName":"spin_result","_valueToSum":400,"fb_currency":"GBP","additional_bet_index":0,"battle_id":"N/A","bet_amount":9,"bet_level":1,"bet_multiplier":1,"coin_balance":9466052,"current_module_name":"CasinoGameModule","days_in_game":0,"extra_param":"N/A","fb_profile":"0","featureGameType":"N/A","game_fps":30,"game_id":"fireEagleBase","game_name":"FireEagleSlots","gem_balance":0,"graphicsQuality":"HD","idfa":"2bf99787-33d2-4ae2-a76a-c49672f97252","internetReachability":"ReachableViaLocalAreaNetwork","isLowEndDevice":"False","is_auto_spin":"False","is_turbo":"False","isf":"False","ishighroller":"False","jackpot_win_amount":90,"jackpot_win_type":"Silver","level":6,"lifetime_gem_balance":0,"no_of_spin":1,"player_total_battles":0,"player_total_shields":0,"start_date":"2019-08-01","total_payments":0,"tournament_id":"T1561970819","userId":"c82cbdff-e5be-4009-ac78-cdeea09ab4b1","versionSessionCount":2,"win_amount":0,"fb_content_id":["123","345","567"]}]',
```
Comment (koladilip) [inline test/integrations/destinations/fb/dataDelivery/business.ts:106]: repeated

File: test/integrations/destinations/fb/dataDelivery/business.ts [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        body: generateProxyV1Payload({
+          endpoint: `https://graph.facebook.com/${VERSION}/RudderFbApp/activities?access_token=my_access_token`,
+          headers: {
+            'x-forwarded-for': '1.2.3.4',
+          },
+          params: {
+            destination: 'fb',
+          },
+          FORM: {
+            event: 'CUSTOM_APP_EVENTS',
```
Comment (koladilip) [inline test/integrations/destinations/fb/dataDelivery/business.ts:95]: is this repeated

File: test/integrations/destinations/fb/dataDelivery/business.ts [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 400,
+            message:
+              "Object with ID 'PIXEL_ID' / 'DATASET_ID' / 'AUDIENCE_ID' does not exist, cannot be loaded due to missing permissions, or does not support this operation",
+            statTags: {
```
Comment (koladilip) [inline test/integrations/destinations/fb/dataDelivery/business.ts:241]: use common stats

File: test/integrations/destinations/fb/dataDelivery/other.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 429,
+            message: 'API User Too Many Calls',
+            statTags: {
```
Comment (koladilip) [inline test/integrations/destinations/fb/dataDelivery/other.ts:42]: use common stats

File: test/integrations/destinations/fb/dataDelivery/business.ts (hunk 1 above)
Comment (sandeepdsvs) [reply test/integrations/destinations/fb/dataDelivery/business.ts]: addressed

File: test/integrations/destinations/fb/dataDelivery/business.ts (hunk 3 above)
Comment (sandeepdsvs) [reply test/integrations/destinations/fb/dataDelivery/business.ts]: addressed

File: test/integrations/destinations/fb/dataDelivery/business.ts (hunk 2 above)
Comment (sandeepdsvs) [reply test/integrations/destinations/fb/dataDelivery/business.ts]: addressed

File: test/integrations/destinations/fb/dataDelivery/other.ts (hunk 4 above)
Comment (sandeepdsvs) [reply test/integrations/destinations/fb/dataDelivery/other.ts:35]: addressed

**Review by koladilip — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**